src/k-provider/ott/provider-parser.js

```diff
@@ -22,8 +22,10 @@ const MediaTypeCombinations: {[mediaType: string]: Object} = {
   [KalturaAsset.Type.MEDIA]: {
     [KalturaPlaybackContext.Type.TRAILER]: () => ({type: MediaEntry.Type.VOD}),
     [KalturaPlaybackContext.Type.PLAYBACK]: mediaAssetData => {
-      if (parseInt(mediaAssetData.externalIds) > 0 || mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
+      if (parseInt(mediaAssetData.externalIds) > 0) {
         return {type: MediaEntry.Type.LIVE, dvrStatus: 0};
+      } else if (mediaAssetData.objectType === LIVE_ASST_OBJECT_TYPE) {
+        return {type: MediaEntry.Type.LIVE, dvrStatus: mediaAssetData.enableTrickPlay ? 1 : 0};
```

Comment:
@RoyBregman suggest to add enum for this (can't remember why we didn't do it).

Reply:
```js
DvrStatus.ON
DvrStatus.OFF
```

Reply:
done

Reply:
@RoyBregman but need also to assign the enum 😁

Reply:
Oh, i thought only declare it ;)
I pushed the wrong file... pushed it now